config/features_config.yml

```diff
@@ -0,0 +1,10 @@
+read_from: 'local'
```

Comment:
Probably more of personal thing but might prefer to see at least which files each parameter belongs to.